ArjunaJTA/jee_transactional_app/src/test/java/org/jboss/narayana/quickstarts/TestManagedBeanCustomerManager.java

```diff
@@ -59,10 +61,8 @@ public static WebArchive createDeployment() {
 				.addAsWebInfResource(EmptyAsset.INSTANCE, "beans.xml");
 		archive.delete(ArchivePaths.create("META-INF/MANIFEST.MF"));
 
-		// Need to make sure we add the arquillian-service and msc as a
-		// dependency
 		final String ManifestMF = "Manifest-Version: 1.0\n"
-				+ "Dependencies: org.jboss.modules,org.jboss.msc,org.jboss.jts\n";
+				+ "Dependencies: org.jboss.jts\n";
```

Comment:
Can I just check, is this a tidy up bit or is it required for the fix? Happy to have it but if it is not required for JBTM-3176 please can you separate it to a different commit (no specific issue number required as it is in the quickstarts but please say something like "Clean-up during JBTM-3176" so we can track it but separate to the core fix)

Reply:
Yes, it's only tidy up the dependencies. When I start investigating the quickstart it seemed to me strange that there is dependency defined to `org.jboss.modules` which I think is pretty rare to be defined. Thus I removed that for tidying things up.

Reply:
OK, so please do split this into a separate commit and please do raise the WFLY - thanks again!